src/components/navbar/NavbarDropdown.vue

```diff
@@ -56,18 +58,31 @@ export default {
         collapsible: Boolean
 
     },
+
     data() {
         return {
             newActive: this.active,
             isHoverable: this.hoverable,
             _isNavbarDropdown: true // Used internally by NavbarItem
         }
     },
+
+    computed: {
+        isShowDropDownItems() {
```

Comment:
It's not really correct because you are checking mobile device and not mobile resolution. Navbar CSS checks the with my media query